index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,500;0,600;0,900;1,200;1,300;1,400;1,600;1,700;1,800&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="white">
+        <header class="head-section">
+          <a class="logo" href="#">My logo</a>
+         <nav>
+          
+           <ul id="list-items">
+                <li><a href="#">Portifolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+          <img src="./images/union-img.png" alt="union-icon" class="union-icon">
+         </nav>
+        </header>
+      <div class="bio">
+        <h1>I'm Magembe Glad to see you!
+        </h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2>Let's connect</h2>
+        <ul class="icons">
+           <li>
+            <a href="#"><img class="social-icons" src="./images/twitter.png" alt="twitter"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/linkkedin.png" alt="linkedin"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/more.png" alt="more"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/git.png" alt="github"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/peace.png" alt="peace"></a>
+           </li>
+        </ul>
+      </div>  
```

Comment:
I think you should be able to achieve the  design by adding a section that will be a wrapper for this bio part, you can give it a class name "main-container" and give it this style 

```
.main-container {
  width: 100%;
  height: 100%;
  z-index: 999;
}
```
Your design presently is not aligned with the Figma design... 😞 

| Your Design | Figma Design |
| :---:   | :---: | 
| <img width="375" alt="image" src="https://user-images.githubusercontent.com/73854999/220939128-f58d535c-f33b-4fae-857b-310af1f31467.png">|<img width="375" alt="image" src="https://user-images.githubusercontent.com/73854999/220939370-b7dd1b13-9953-4f03-8897-cd3f18b553b9.png">|